source/win-main/EditorBranch.vue

```diff
@@ -198,24 +181,24 @@ body .split-pane-container {
   display: flex;
   width: 100%;
   height: 100%;
-  position: relative;
 
   .resizer {
-    position: absolute;
     z-index: 1;
     transition: background-color 0.5s ease;
-  }
 
-  .resizer:hover { background-color: rgba(128, 128, 128, .5); }
+    &:hover { background-color: rgba(128, 128, 128, .5); }
+  }
 
   .resizer.horizontal {
```

Comment:
We can move this inside the main `.resizer`.

Reply:
It already is inside the main `.resizer`, I believe …?

Reply:
If I'm not wrong, it's outside of the main `.resizer`. I saw your change with the `hover` a few lines above, that's how the `.resizer.horizontal` (and `.resizer.vertial`) caught my attention.

Reply:
Ahhhhh, now I see what you meant: I looked at the changed line, not the one you actually commented on 😂🙈 Yeah, I see your point but I think it's fine for now ... feel free to change that if you think it's nicer that way!